streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -291,8 +297,9 @@ public void shouldSuccessfullyStartWhenLoggingDisabled() throws InterruptedExcep
         assertTrue(startupLatch.await(30, TimeUnit.SECONDS));
     }
 
-    @Test
-    public void shouldProcessDataFromStoresWithLoggingDisabled() throws InterruptedException {
+    @ParameterizedTest
```

Comment:
Could we just parameterize the whole class than parameterizing each test method?

Reply:
In Junit 5 it is apparently not possible to parametrize on test class level. However, you can use a method as the source for the parameter values `true` and `false` for all test methods instead of enumerating `true` and `false` for each test method. I used the method source.

Reply:
Ack, thanks.